assets/interfaces.go

```diff
@@ -267,6 +268,24 @@ type TemplateTranslation interface {
 	Channel() ChannelReference
 }
 
+// TicketerUUID is the UUID of a ticketer
+type TicketerUUID uuids.UUID
+
+// Ticketer is a system which can open or close tickets
+//
+//   {
+//     "uuid": "37657cf7-5eab-4286-9cb0-bbf270587bad",
+//     "name": "Support Tickets",
+//     "type": "mailgun"
+//   }
+//
+// @asset ticketer
+type Ticketer interface {
```

Comment:
<img width="668" alt="Captura de Pantalla 2020-04-23 a la(s) 16 23 25" src="https://user-images.githubusercontent.com/675558/80150910-d094af00-857e-11ea-9f30-c1096f9af6c3.png">

Reply:
I'm not going to fight this super hard but I think this is less clear than TicketingService, especially because "Ticketer" evokes a person, which this most definitely is not.

Reply:
Service has a meaning in goflow already.. if we need a service for Tickets.. then that's the TicketService.. so it can't also be the name of the asset

Reply:
Open to suggestions here but we can't pass TicketService assets to the TicketService